Kernel/Graphics/GraphicsManagement.cpp

```diff
@@ -167,6 +168,10 @@ UNMAP_AFTER_INIT bool GraphicsManagement::determine_and_initialize_graphics_devi
             dmesgln("Graphics: Using VirtIO console");
             adapter = VirtIOGraphicsAdapter::initialize(device_identifier);
             break;
+        case PCI::VendorID::VMWare:
+            dmesgln("Graphics: Using VirtIO console");
```

Comment:
That's a lie :)

(perhaps we don't need a log message here, or if you'd like to have one, let's add one to all branches above)

Reply:
Oops. my bad